src/test/java/task/TodoTaskTest.java

```diff
@@ -0,0 +1,42 @@
+package task;
+
+import exceptions.DukeException;
+import org.junit.jupiter.api.Test;
```

Comment:
Not sure if it is necessary, maybe you can consider to separate the import statements from different packages? Please do check if this is correct.
I have also noticed this in the other test files.

Reply:
Great Point! Will push this out in the next commits!